procedures/release-duty/index.rst

```diff
@@ -340,30 +331,31 @@ No, after b1, there isn't signoff from RelMan on DevEdition builds. QA only veri
 weeks. With the exception of b1, and assuming all the tasks complete as expected, the DevEdition builds should be
 shipped at the same time as we receive signoff for the corresponding desktop builds.
 
-7. *How should I inform the ReleaseDuty team of recent changes in automation that may impact an upcoming release?*
-
-You can mention it to the current ReleaseDuty folks in the #releaseduty channel. Please also add it to the upcoming
-release in the ../releases/FUTURE/ dir. See `future release support <../releases/FUTURE/README.md>`__ for more details.
-
-8. *How do I coordinate with marketing on release day?*
+7. *How do I coordinate with marketing on release day?*
 
 Join the #release-coordination channel on Mozilla Slack
```

Comment:
I haven't done this, but we can keep it for now probably. This might have been for releases with major marketing pushes associated with them.